src/components/App/ScheduleLoginManager.js

```diff
@@ -0,0 +1,38 @@
+import React, { PureComponent } from 'react';
+import { checkUser, loadUser, loadSchedule } from '../../actions/AppStoreActions';
+import { LoadSchedule, SaveSchedule } from './LoadSaveFunctionality';
+import AccountBase from './AccountBase';
+
+class ScheduleLoginManager extends PureComponent {
+    state = {
+        user: null,
+    };
```

Comment:
Convert to functional component
```suggestion
const ScheduleLoginManager = (props) => {
    const [user, setUser] = useState(null);
```